### PR TITLE
Lm aw 140 add anim and error message

### PR DESCRIPTION
**Pull request by LarryMatte**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -66,6 +65,13 @@
             <version>${htmlunit.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.vectomatic</groupId>
```
Comment (christiangoudreau) [inline pom.xml]: Ohhh, I don't like that library, lol! (Had to fix it in the past for another client :D)

**Comment by christiangoudreau** [comment]
Screen?

**Comment by LarryMatte** [comment]
screen shot 2015-05-22 at 2 03 10 pm

**Comment by christiangoudreau** [comment]
LGTM

**Comment by jasonlemay** [comment]
@LarryMatte LGTM, please merge when you come back

**Pull request merged**